packages/bottender-dialogflow/src/types.ts

```diff
@@ -18,8 +18,8 @@ export type QueryResult = {
 
 export type Message = {
   platform: Platform;
-  text: string[];
-};
+  message: string;
```

Comment:
Is there any reference for this change?

Reply:
https://cloud.google.com/dialogflow/docs/reference/rest/v2/projects.agent.intents#Message

Reply:
It look likes a undocumented field.